shared/test/test_sources.rb

```diff
@@ -0,0 +1,49 @@
+require 'minitest/autorun'
+require 'rack/test'
+require File.expand_path '../../../deployment', __FILE__
+require File.expand_path '../../middleware/assets_api', __FILE__
```

Comment:
looks like you are testing the assets API. this is concerning since presumably the versioning-related tests are passing, despite versioning not being enabled on the assets s3 bucket?